src/render/PointArray.cpp

```diff
@@ -368,7 +368,7 @@ bool PointArray::loadFile(QString fileName, size_t maxPointCount)
 
     // Sort points into octree order
     emit loadStepStarted("Sorting points");
-    std::unique_ptr<size_t[]> inds(new size_t[m_npoints]);
+    std::unique_ptr<size_t[]> inds = std::unique_ptr<size_t[]>(new size_t[m_npoints]);
```

Comment:
It *was* churned (deleted and re-added later when I changed what I was doing). Consider it unchurned.